[2017-03-29 17:44:35] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:44:35] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:44:36] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "xm+YTJahhwRUy9BAffuqq21fDlxtMtyNepZ9T\/Q3VlMDoJgvrV1AZKq73MsisjembzTENFCuCoojFJjdtR2MSQ==", "8364819a94893d8d039d8a10c4c92407", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"xm+YTJahhwRUy9BAffuqq21fDlxtMtyNepZ9T\\/Q3VlMDoJgvrV1AZKq73MsisjembzTENFCuCoojFJjdtR2MSQ==\", \"8364819a94893d8d039d8a10c4c92407\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 17:45:19] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:45:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:45:19] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "SX7UHA8w5iZWd6KSN11\/vCx0B56sheeLDPzqR8Jh1oX4itJJ9cwS\/RA87FG\/yeCo1UhKlvIVA5YUuZfcMMT8Ew==", "febc93c60147c9b907917882a6622ef9", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"SX7UHA8w5iZWd6KSN11\\/vCx0B56sheeLDPzqR8Jh1oX4itJJ9cwS\\/RA87FG\\/yeCo1UhKlvIVA5YUuZfcMMT8Ew==\", \"febc93c60147c9b907917882a6622ef9\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 17:45:39] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:45:39] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:45:40] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "saf3VraiO1BECz0kWX8v3KrsmQZDZJJCgWgnq6frEcVNYOAHwIKR\/SscgV2NjkZnIxFxwiNY103+\/vXd9+AEBg==", "397f275ce93b9445a0a64344e64d9d2b", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"saf3VraiO1BECz0kWX8v3KrsmQZDZJJCgWgnq6frEcVNYOAHwIKR\\/SscgV2NjkZnIxFxwiNY103+\\/vXd9+AEBg==\", \"397f275ce93b9445a0a64344e64d9d2b\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 17:52:29] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:52:29] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:52:29] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "L6PYnsu7pSu79D1gZJF3TlyTHfe93vyqZQ2nThgXPuYBVNkciJUB31eK4xkJ7B46upqCi8o0VrkoVj3Gov9Dcg==", "eb42489fca28d481a0760b760a093052", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"L6PYnsu7pSu79D1gZJF3TlyTHfe93vyqZQ2nThgXPuYBVNkciJUB31eK4xkJ7B46upqCi8o0VrkoVj3Gov9Dcg==\", \"eb42489fca28d481a0760b760a093052\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 17:53:09] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:53:09] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:53:10] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "dBLG0618rpScTfzLeadfy6QartjGS4kI0PNpTSUBjiTl+zp885Jqxms6lBgXh5hCoECI+KQkC7WBlhijaY+ryA==", "f0a437b17b354d4078517b0caa2d0e18", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"dBLG0618rpScTfzLeadfy6QartjGS4kI0PNpTSUBjiTl+zp885Jqxms6lBgXh5hCoECI+KQkC7WBlhijaY+ryA==\", \"f0a437b17b354d4078517b0caa2d0e18\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 17:54:00] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 17:54:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 17:54:01] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["Juan", "Obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "LPdP+HB4oKkEy7fBJPwQFrcBAn2IMQvTK7wUAiTA+mK2PXPJ+Ik6Wy5hPRKTFQRdKCRz2eAJAU3y0HdhuYJpZA==", "f6fb0ae5a0bc9c3fedf005516a1dd7e3", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"Juan\", \"Obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"LPdP+HB4oKkEy7fBJPwQFrcBAn2IMQvTK7wUAiTA+mK2PXPJ+Ik6Wy5hPRKTFQRdKCRz2eAJAU3y0HdhuYJpZA==\", \"f6fb0ae5a0bc9c3fedf005516a1dd7e3\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 18:01:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:02:35] request.INFO: Matched route "sitio_contacto". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::contactoAction","_route":"sitio_contacto"},"request_uri":"http://localhost/moveLanguage/web/sitio/contacto/"} []
[2017-03-29 18:02:35] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 18:03:06] app.ERROR: Exception occurred while flushing email queue: Connection to mail.safabox.com:25 Timed Out [] []
[2017-03-29 18:05:52] request.INFO: Matched route "test_nivel". {"route_parameters":{"_controller":"TestBundle\\Controller\\DefaultController::nivelAction","_route":"test_nivel"},"request_uri":"http://localhost/moveLanguage/web/test/nivel/"} []
[2017-03-29 18:05:52] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 18:06:22] app.ERROR: Exception occurred while flushing email queue: Connection to mail.safabox.com:25 Timed Out [] []
[2017-03-29 18:07:03] request.INFO: Matched route "sitio_registrar". {"route_parameters":{"_controller":"SitioBundle\\Controller\\DefaultController::registrarAction","_route":"sitio_registrar"},"request_uri":"http://localhost/moveLanguage/web/sitio/registrar/"} []
[2017-03-29 18:07:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-29 18:07:03] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["facundo", "obregon", "facundo.obregon@safabox.com", "2017-03-29", "Activo", "yZQgaY2DFhGrcx3WuCwT5YUHW0I2RK68NnAzCW4MD9tWTC7P8jvr5kvDT\/+ni23jozN4Ptr9x1PGFm6u7+5e8w==", "b7009dc178c0b61346d25f0d8f7bacc9", null, null, null, 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74'" at C:\wamp\www\moveLanguage\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 66 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO user (name, last_name, email, starting_date, status, password, salt, country, province, city, allows_email, contact_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"facundo\", \"obregon\", \"facundo.obregon@safabox.com\", \"2017-03-29\", \"Activo\", \"yZQgaY2DFhGrcx3WuCwT5YUHW0I2RK68NnAzCW4MD9tWTC7P8jvr5kvDT\\/+ni23jozN4Ptr9x1PGFm6u7+5e8w==\", \"b7009dc178c0b61346d25f0d8f7bacc9\", null, null, null, 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:66, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'facundo.obregon@safabox.com' for key 'UNIQ_8D93D649E7927C74' at C:\\wamp\\www\\moveLanguage\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-29 18:27:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /inscripciones"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /inscripciones\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:27:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /inscripcion/"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /inscripcion/\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:28:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:28:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:28:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-29 18:30:42] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:256)"} []
[2017-03-31 16:55:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/aos.js.map"" at C:\wamp\www\moveLanguage\app\cache\prod\classes.php line 2599 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/aos.js.map\" at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\classes.php:2599, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\moveLanguage\\app\\cache\\prod\\appProdUrlMatcher.php:264)"} []
